README.md

```diff
@@ -178,14 +178,12 @@ The browser will add a new request header with the resulting signature over a co
 
 ```
 Sec-Signature:
-  alg=<alg>
-  public-key=<pk>
-  sig=<signature>
+  signatures=[(<issuer>, {"alg": "ecdsa_secp256r1_sha256", "public-key": <pk>, "sig": <signature>}), ...]
   sign-request-data=<include, headers-only>
 ```
 
 
-The canonical CBOR data (verifiable by the signature) should be computable from a request, and so does not need to be sent over the wire from the browser. The `Signed-Headers` header, and the value of `sign-request-data` should be enough to re-construct it server side, robust to things like header re-ordering, etc. In Version 3 of the protocol, the algorithm used for signing is [`ecdsa_secp256r1_sha256`](https://tools.ietf.org/html/rfc8446#section-4.2.3).
+The canonical CBOR data (verifiable by the signature) should be computable from a request, and so does not need to be sent over the wire from the browser. The `Signed-Headers` header, and the value of `sign-request-data` should be enough to re-construct it server side, robust to things like header re-ordering, etc. A key for each requested issuer is included in the signatures field as a Structured Headers list of pairs of issuers and their respective signature material (algorithm, key, signature). In Version 3 of the protocol, the algorithm used for signing is [`ecdsa_secp256r1_sha256`](https://tools.ietf.org/html/rfc8446#section-4.2.3).
```

Comment:
> Structured Headers list

I'd prefer leaving the details of the fields' encoding out: perhaps we could just use the word "collection" instead of the quoted phrase.

(For instance, I don't know if there's a Structured Headers notion of a "pair." It makes life a little easier if we're able to keep the examples as simple-as-possible pseudocode that shows the logical structure but doesn't try to look like the wire format. It also keeps the explainer firmly in "high-level illustration of potential functionality" territory and reduces the risk it can be misconstrued as normative.)